Marlin/src/pins/pins.h

```diff
@@ -532,6 +532,8 @@
   #include "stm32/pins_STEVAL_3DP001V1.h"       // STM32F4                                env:STM32F401VE_STEVAL
 #elif MB(BIGTREE_SKR_PRO_V1_1)
   #include "stm32/pins_BTT_SKR_PRO_V1_1.h"      // STM32F4                                env:BIGTREE_SKR_PRO
+#elif MB(BIGTREE_GTR_V1_0)
+  #include "stm32/pins_BTT_GTR_V1_0.h"          // STM32F4                                env:BIGTREE_GTR
```

Comment:
The pins file is missing. Please supply a new one.

Reply:
> The pins file is missing. Please supply a new one.
Actually, I have uploaded the pin file. I don't know why I can't pass it. It's a little strange

Reply:
Not a problem to do it as a separate PR. We'll figure it all out!

Reply:
Hey, it looks like the pins file is here. Thanks!